backend/app/routers/flows.py

```
@@ -0,0 +1,753 @@
+            # Create result record
+            flow_result = DigiFlowResult(
+                flow_id=flow.id,
+                data=extraction_result.get("data", {}),
+                plain_data={"plain_text": content["plain_text"][:5000]},
+                text_blocks=text_blocks,
+                data_origin=DataOrigin.SYSTEM.value,
+                version=1,
+            )
+            db.add(flow_result)
+
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)